implementations/rust/ockam/ockam_api/src/cloud/project.rs

```diff
@@ -95,6 +95,21 @@ impl Project {
     }
 }
 
+#[derive(Encode, Decode, Serialize, Deserialize, Debug, Default, Clone, Eq, PartialEq)]
```

Comment:
Since this is only a backend-generated response type that's never re-transmitted elsewhere, can we skip Encode and Serialize? @adrianbenavides

Reply:
Exactly, good catch!

Reply:
I dropped them now in the latest commit, let me know if you need them back @adrianbenavides